sensor.js

```diff
@@ -101,21 +101,22 @@ class BME680Plugin {
 					if (data.data.heat_stable) {
 						this.log(`data(temp) = ${ JSON.stringify(data, null, 2) }`);
 
+						const airQuality = computeIAQ(roundInt(data.data.gas_reistance), roundInt(data.data.humidity));
+
 						this.loggingService.addEntry({
 							time: moment().unix(),
 							temp: roundInt(data.data.temperature),
 							pressure: roundInt(data.data.pressure),
 							humidity: roundInt(data.data.humidity),
-							airQuality: roundInt(data.data.gas_resistance)
+							airQuality: airQuality
```

Comment:
Unfortunatelly this is not going  to be enough to fix the problem. Reffering to the `fakegato-history` documentation:

>   this.loggingService.addEntry({time: Math.round(new Date().valueOf() / 1000), temp: this.temperature, humidity: this.humidity, ppm: this.ppm});

PPM here is the actual measurement of parts per million of compounds detected by the sensor. Problem is that the sensor detects gas resistance in ohms and I couldn't really find a way to get from gas resistance to compound PPM.

 I tried to create a workaround by mapping the air quality score that `computeIAQ` creates  to a random value that would yield the same air quality label as described by #2 https://github.com/simont77/fakegato-history/issues/107 but that did not work either. I am still trying to figure out how to get it working